### ISSUE TITLE
Dataset quality

### ISSUE THREAD

**Issue by Dobatymo** [rafalposwiata/depression-detection-lt-edi-2022#4]
I know it's not really an issue with your model/code, but the quality of the original dataset is not that great. I feel there are many mislabeled instanced in `not depression`. I am not a psychologist or psychiatrist, but I think (for example) the following should not be labeled like this

> train_pid_7353	Someone please kill me, please, please, please.... : I can't do this, there is nothing. The only thing I have to work toward are my own goals and I genuinely hate everything about myself so they give nothing if I achieve my goals. Even those I've achieved just feel like nothing. I am done. Please I want to die.	not depression

in `data/original_dataset/train.tsv`.

According to 3.2 Data Annotation in https://arxiv.org/pdf/2202.03047 this should be `Moderately depressed` I guess.

EDIT: Found original repo and opened issue there https://github.com/Kayal-Sampath/detecting-signs-of-depression-from-social-media-postings/issues/3

**Comment by rafalposwiata** [comment]
I agree with you that the quality of this dataset is not very good. Yes, as you mentioned, the problem should instead be directed to the authors of this datasetorganizers of the competition. Thank you for reporting it to them.
